src/typechecker/FStar.TypeChecker.TcInductive.fst

```diff
@@ -127,6 +127,9 @@ let tc_data (env:env_t) (tcs : list (sigelt * universe))
                else raise_error (Errors.Fatal_UnexpectedDataConstructor, ("Unexpected data constructor")) se.sigrng
          in
 
+ 	 (* Typecheck attributes *)
+	 let attrs = tc_local_attributes env attrs in
```

Comment:
Are attributes also checked in both the phases, with verification conditions?

Reply:
(I guess this is the same behavior from current `tc_binder`.)

Reply:
I'm really not familiar with F*'s typechecker, so I'm really not sure, I basically just mimicked what `tc_binder` does